Firestore/Source/Local/FSTLevelDBQueryCache.mm

```diff
@@ -43,6 +45,19 @@
 using leveldb::Status;
 using firebase::firestore::model::DocumentKeySet;
 
+namespace {
+
+FSTListenSequenceNumber ReadSequenceNumber(const absl::string_view &slice) {
+  FSTListenSequenceNumber decoded;
+  absl::string_view tmp(slice.data(), slice.size());
+  if (OrderedCode::ReadSignedNumIncreasing(&tmp, &decoded)) {
```

Comment:
Super-ultra-mega-nitty-mcnitface: elsewhere we largely follow the pattern of

```
if (is-a-failure) { 
  return handle_failure();
}
proceed();
```
Here it doesn't matter now buf if we ever added anything to the affirmative case it would look weird. Consider reversing the clauses. (Feel free to ignore.)

Reply:
Done.